js/components/lightbox.js

```
@@ -0,0 +1,312 @@
+            const lightbox = document.getElementById(this.lightboxId);
+
+            const img = lightbox.querySelector('.lightbox-image');
+            const title = lightbox.querySelector('.lightbox-title');
+            const counter = lightbox.querySelector('.lightbox-counter');
+
+            // Update image - check if the original gallery item has an updated src
+            const galleryItem = document.querySelector(`[data-index="${this.currentIndex}"] .lazy-image`);
+            let imageSrc = item.viewUrl || item.src;
+            
+            // If the gallery item has a successfully loaded image, use that src
+            if (galleryItem && galleryItem.src && !galleryItem.src.includes('data:image')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)